pkg/server/server.go

```diff
@@ -445,8 +445,9 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 				})
 			if err := backend.Send(pkt); err != nil {
 				klog.ErrorS(err, "DIAL_REQ to Backend failed", "serverID", s.serverID)
+			} else {
+				klog.V(5).Infoln("DIAL_REQ sent to backend") // got this. but backend didn't receive anything.
```

Comment:
If we're going to add this, it probably makes sense to also pass through the serverID and random. Also makes me think we should consider surfacing a backend/agent id.

Reply:
I don't disagree with this, but this log line already exists (see line below), this is just moving it into an else block so we don't log this on error.